Misc/Adam.cpp

```diff
@@ -3354,3 +3354,24 @@ int AdamInit()
 
 	return 1;
 }
+
+void WriteAlteredConfigVarsToIni()
+{
+	const char* alteredConfigVars[] = {
+		"projmetroen",
+		"relativeedges",
+		"itemclickmovecurs",
+		"itemtimelock"
```

Comment:
This saves the current project timebase as the default for new projects without asking. `itemtimelock` is a project configvar.

Reply:
Thanks. Hm...so the only one that should be made persistant is `itemclickmovecurs`, right?
Should I then rather revert the `WriteAlteredConfigVarsToIni()` and write to ini in the functions itself which alter it? (7 functions if I counted correctly.) This would also prevent the writing to ini at exit when actually nothing has changed. On the other hand, it would write every time when toggled and not just one time at exit.

Reply:
> Should I then rather revert the WriteAlteredConfigVarsToIni() and write to ini in the functions itself which alter it? (7 functions if I counted correctly.)

I think a `PersistItemClickMoveCurs` called from the 7 actions makes sense. Many of REAPER's own options set/toggle actions immediately write to the .ini.

Writing on exit might conflict with REAPER or another extension/script wanting to save a conflicting value (possibly from user interaction made after those SWS actions were used).

Reply:
If going the write immediately route, can you cherry-pick fb46dd66e15ec06032dbcb95be4647019f2967b8 first and add the persist fix on top? (Then I'll close #1446 in favor of this PR.)

Reply:
I'm AFK now for a couple of hours. If you want to merge this in 2.12.1.1 fast (?), don't hesitate to go ahead and do it.
Otherwise I'd try when I'm back. (I like the idea of adding a `.persist()` to `configvar.h`, if I understood the idea correctly.)

Reply:
There's no hurry, but I have some free time now, so I'll give it a go.